fonts/generate.py

```diff
@@ -211,7 +211,7 @@ def generate_css(opts: Namespace) -> bool:
     Et.register_namespace("", SVG_NS["svg"])
     svg_font: Et.ElementTree = Et.parse(str(font_pth))
     font_el: Optional[Et.Element] = svg_font.find(".//svg:defs/svg:font", SVG_NS)
-    if not font_el:
+    if font_el is None:
```

Comment:
I would caution against making such changes. The XML libraries are weird about what constitutes “falsey” and what is actually `None`. If you don’t need to make these sorts of changes and are just doing it for your own sense of aesthetics, I would leave them.

Reply:
Without this change Python throws: 

> DeprecationWarning: Testing an element's truth value will raise an exception in future versions.  Use specific 'len(elem)' or 'elem is not None' test instead.

And in doubt it may help to consult the documentation: we use `find` from `xml.etree.ElementTree` which returns `None` if nothing is found. 
See <https://docs.python.org/3/library/xml.etree.elementtree.html#xml.etree.ElementTree.Element.find>.

Reply:
Ok, great! Thanks